packages/hardhat-toolbox/package.json

```diff
@@ -33,21 +33,21 @@
     "LICENSE",
     "README.md"
   ],
-  "dependencies": {},
   "devDependencies": {
     "@ethersproject/abi": "^5.4.7",
     "@ethersproject/providers": "^5.4.7",
-    "@nomicfoundation/hardhat-network-helpers": "^1.0.0",
     "@nomicfoundation/hardhat-chai-matchers": "^1.0.0",
+    "@nomicfoundation/hardhat-network-helpers": "^1.0.0",
     "@nomiclabs/hardhat-ethers": "^2.0.0",
     "@nomiclabs/hardhat-etherscan": "^3.0.0",
     "@typechain/ethers-v5": "^10.1.0",
     "@typechain/hardhat": "^6.1.2",
     "@types/chai": "^4.2.0",
     "@types/mocha": ">=9.1.0",
     "@types/node": "^14.0.0",
-    "@typescript-eslint/eslint-plugin": "4.29.2",
-    "@typescript-eslint/parser": "4.29.2",
+    "@typescript-eslint/eslint-plugin": "5.53.0",
```

Comment:
Had to update these dependencies (`@typescript-eslint/eslint-plugin` and `@typescript-eslint/parser`) from `4.29.2` to `5.53.0` as the other packages cause the old version does not support `ignoreTernaryTests`.

That broke the `@typescript-eslint/experimental-utils` import but I feel like I I'm just monkey patching by installing it here to make it work:

```bash
$ npx yarn lint
 | Error: Failed to load plugin '@nomiclabs/eslint-plugin-hardhat-internal-rules' declared in '.eslintrc.js » ../packages/hardhat-toolbox/../../config/eslint/eslintrc.js': Cannot find module '@typescript-eslint/experimental-utils'
 | Require stack:
 | - ../packages/eslint-plugin/onlyHardhatErrorRule.js
 | - ../packages/eslint-plugin/index.js```

Reply:
Oh, I didn't realize it was going to be that much of a hassle. Don't worry, I opened https://github.com/NomicFoundation/hardhat/issues/4028 to remember to do that at some point.